### ISSUE TITLE
Is there a Python/Pytorch version of the loss function?

### ISSUE THREAD

**Issue by sarmientoj24** [KIuML/DependenceAwareMLCLoss#1]
Came across with your great work. However, I would like to know if there is a Python version of this. 
Or something easier to port  or use.

Thank you

**Comment by mwever** [comment]
Thanks for your interest. Our implementations are in Java only, so unfortunately there is no Python version of our concrete instantiation of loss functions. Admittedly, our implementation is also quite deeply embedded in the library we used (AILibs). We will think about providing a Python implementation on top of this - should not be too hard to do.
